privacyidea/lib/resolvers/LDAPIdResolver.py

```diff
@@ -983,14 +996,22 @@ def testconnection(cls, param):
         size_limit = int(param.get("SIZELIMIT", 500))
         serverpool_rounds = int(param.get("SERVERPOOL_ROUNDS") or SERVERPOOL_ROUNDS)
         serverpool_skip = int(param.get("SERVERPOOL_SKIP") or SERVERPOOL_SKIP)
-        if is_true(param.get("TLS_VERIFY")) \
-                and (ldap_uri.lower().startswith("ldaps") or
-                                    param.get("START_TLS")):
-            tls_version = int(param.get("TLS_VERSION") or ssl.PROTOCOL_TLSv1)
-            tls_ca_file = param.get("TLS_CA_FILE") or DEFAULT_CA_FILE
-            tls_context = Tls(validate=ssl.CERT_REQUIRED,
-                              version=tls_version,
-                              ca_certs_file=tls_ca_file)
+        if ldap_uri.lower().startswith("ldaps") or param.get("START_TLS"):
```

Comment:
We have the same lengthy code to determine the TLS context in the LDAP resolver and in testconnection.

Do you think we can simplify this with a method like

     get_tls_context(...alll parameters to create the tls context...)

I am not sure myself, since the parameter list would probably get very lengthy.